### ISSUE TITLE
[Feature request] Show the entire franchise when I only watched one item

### ISSUE THREAD

**Issue by HelewisRyuu** [gubo97000/aniCheck#5]
Hi, first, I would like to thank you for this because it helps me a lot.

I usually choose what I'm going to finish based on the 'Missing' section ordered by 'Time To Complete', but some shows show up as having only one title to finish, when it actually has other seasons as well.
So, I would like to request, if possible, to show all the things linked to the franchise in only one item.

For example, the 'Nanatsu no Taizai' franchise. I only watched the first season a long time ago, and now, I'm watching the new anime 'Nanatsu no Taizai: Mokushiroku no Yonkishi'.
They show up as 2 different items because of the gap between seasons. I know that when I watch them, they will be united, but I wanted it would be easier to know beforehand the size of the franchise I still need to finish.
![image](https://github.com/gubo97000/aniCheck/assets/161946123/c55a2c4d-bc0b-457d-b7d2-d03c302a7fda)

**Comment by gubo97000** [comment]
Yeah I know the pain, I did not implement it yet because I need to think of the best way to handle the fetching of the missing elements; the amount of api calls that needs to be done for the whole catalog is pretty big and would probably be blocked by anilist. Another problem is that I need to find a intelligent way to have this information cached without introducing cache related bugs... But yeah is on the list of feature.

I am sorry I am not working much on aniCheck, but I am bad at organizing between work thesis and side projects :D
